packages/berry-core/sources/scriptUtils.ts

```diff
@@ -16,7 +16,8 @@ async function makePathWrapper(location: string, name: string, argv0: string, ar
   if (process.platform === `win32`) {
     await xfs.writeFilePromise(`${location}/${name}.cmd`, `@"${argv0}" ${args.join(` `)} %*\n`);
   } else {
-    await xfs.writeFilePromise(`${location}/${name}`, `#!/usr/bin/env bash\nexec "${argv0}" ${args.map(arg => `'${arg.replace(/'/g, `'"'"'`)}'`).join(` `)} "$@"\n`);
+    await xfs.writeFilePromise(`${location}/${name}`, `#!/usr/bin/env bash\nexec "${argv0}" ${args.map(arg => `'${arg.replace(/'/g, `'"'"'`)}'`)
+      .join(` `)} "$@"\n`);
```

Comment:
Did something change here?

Reply:
Nothing changes here, just my IDE changing the format 🙂